notebooks/00_quick_start/sar_aml_compute_movielens.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>I don't understand why we are using papermill here, we are not using a notebook. Can you please elaborate?</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Removed papermill reference now